plugins/snmp_receiver_plugin/snmp_server/unregister_switches.py

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright © 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+#
+# @author: Alexander Tolikin
+# @date: 30 January, 2024
+#
+
+import helpers
+from resources import Switch, UFMResource
+
+if __name__ == "__main__":
```

Comment:
Please add try...except clause for a better error handling.

Reply:
added